userguide/tutorials/stripe_plugin.adoc

```diff
@@ -1,3 +1,5 @@
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/tutorials/39680228343_69894dd3a1_c.jpgg[align=center]
```

Comment:
Extension `jpgg` seems weird, typo?

Also, could you provide a name a bit more descriptive for the image -- instead of `39680228343_69894dd3a1_c` ?